docs/ref/unicode.txt

```diff
@@ -48,7 +47,7 @@ General string handling
 
 Whenever you use strings with Django -- e.g., in database lookups, template
 rendering or anywhere else -- you have two choices for encoding those strings.
-You can use normal Unicode strings or bytestrings (starting with a 'b').
+You can use normal strings or bytestrings (starting with a 'b').
 
```

Comment:
Yes, this whole page needs rewriting. It isn't just a matter of updating some sentences; the basic premise of the page changes when str == unicode instead of str == bytes.

I created a ticket about that this morning before I noticed this PR. I think it's best to leave out changes to this file from this PR and do them in a dedicated PR.